### PR TITLE
Change ci platform to Github Actions

### PR DESCRIPTION
**Pull request by atm-snag2**
# What

- Add github actions config to run test
- Remove travis config
- Add Environment variable to use `bundle` in test

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Test
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/test.yml'
+
+permissions: read-all
```
Comment (mziyut) [inline .github/workflows/test.yml]: i think to giving too many permissions for CI execution

```suggestion
permissions:
  contents: read
```

[Workflow syntax for GitHub Actions - GitHub Docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions)

**Review by mziyut — COMMENTED** [review]
Added a comment!
plz update CHANGELOG.md :pray:

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: Test
+
+on: ['push', 'pull_request']
```
Comment (mziyut) [inline .github/workflows/test.yml]: I think to this test should be triggered by ...

- New commit to default branch (main or master branch)
- Create pull request

```suggestion
on:
  pull_request:
  push:
    branches:
      - main
      - master
```

**Review by mziyut — COMMENTED** [review]
Additional commented. sorry....

**Review by mziyut — APPROVED** [review]
thanks, LGTM!

**Review by getty104 — APPROVED** [review]

**Comment by atm-snag2** [comment]
I lost adding dependencies. re-review please.

**Review by getty104 — APPROVED** [review]
LGTM!

**Review by takashi-k-bird — APPROVED** [review]
LGTM!

**Review by mziyut — APPROVED** [review]

**Pull request merged**